lib/embed.dart

```diff
@@ -46,7 +46,8 @@ void init(EmbedOptions options) {
   _embed = Embed(options);
 }
 
-enum EmbedMode { dart, flutter, html, inline }
+// ignore: constant_identifier_names
+enum EmbedMode { dart, flutter, html, inline, flutter_showcase }
```

Comment:
once this (hopefully :D) gets merged, it would be nice to mention this new embed mode in the Wiki 👍